app/webpack.config.js

```diff
@@ -158,40 +158,12 @@ module.exports = {
             ],
         }),
         new webpack.EnvironmentPlugin(loadedDotenv),
-        new webpack.EnvironmentPlugin({
-            GIT_VERSION: gitRevisionPlugin.version(),
-            GIT_BRANCH: gitRevisionPlugin.branch(),
-            SENTRY_ENVIRONMENT: process.env.SENTRY_ENVIRONMENT || '',
-            SENTRY_DSN: process.env.SENTRY_DSN || '',
-            VERSION: process.env.VERSION || '',
-        }),
         ...(analyze ? [
             new BundleAnalyzerPlugin({
                 analyzerMode: 'static',
                 openAnalyzer: false,
             }),
         ] : []),
-        ...(process.env.SENTRY_DSN ? [
-            new SentryPlugin({
-                include: dist,
-                validate: true,
-                ignore: [
-                    '.cache',
-                    '.DS_STORE',
-                    '.env',
-                    '.storybook',
-                    'bin',
-                    'coverage',
-                    'node_modules',
-                    'scripts',
-                    'stories',
-                    'test',
-                    'travis_scripts',
-                    'webpack.config.js',
-                ],
-                release: process.env.VERSION,
-            }),
-        ] : []),
```

Comment:
@fonty1 so I put this stuff into the common set of plugins so it can be tested for dev builds. i.e. if you set a `SENTRY_DSN` env var, it will do the sentry stuff without needing to wait for a production build. Were these causing a problem for you in dev?